### PR TITLE
studio: small UI polishes

### PR DESCRIPTION
**Pull request by achen5671**
## Summary
* fix input text being cut off in the toolbar
* fix scene image overflow
* remove extra space in "Add Scene" button
## References
closes #_insert number here_

### CODE REVIEW

**Review by DanielBelmes — APPROVED** [review]

File: packages/ui/src/primitives/tailwind/Input/index.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
 
     const twClassname = twMerge(
       'text-base font-normal tracking-tight',
-      'textshadow-sm flex h-9 w-full rounded-lg border border-theme-primary bg-theme-surfaceInput px-3.5 py-5 transition-colors',
+      'textshadow-sm flex h-9 w-full rounded-lg border border-theme-primary bg-theme-surfaceInput px-2 py-5 transition-colors',
```
Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/Input/index.tsx]: In [native designs](https://www.figma.com/design/Gn8EScrMmg1g6sQBy8f6lz/Hi-FI%3A-Dashboard-%26-Admin?node-id=863-101448&t=CmLbdDLbokTEUFtr-0), the padding is actually 14px
image

Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/Input/index.tsx:82]: The [native designs](https://www.figma.com/design/Gn8EScrMmg1g6sQBy8f6lz/Hi-FI%3A-Dashboard-%26-Admin?node-id=863-101508&t=sPnUqPsUxXN1kAiT-0) in input having a 14px padding.

**Review by aditya-mitra — CHANGES_REQUESTED** [review]
Looks good. Just a few changes.
_(the primitives are used in multiple places - admin, studio, multitenancy. we need to make our changes with respect to figma designs)_

File: packages/ui/src/primitives/tailwind/Input/index.tsx (hunk 1 above)
Comment (achen5671) [reply packages/ui/src/primitives/tailwind/Input/index.tsx:82]: seems like the image above is from the admin dashboard. This pr is for the studio

Comment (achen5671) [reply packages/ui/src/primitives/tailwind/Input/index.tsx]: just want to confirm, the image above is for the admin dashboard. the spacing in the design shows 8px
Screenshot 2024-06-03 at 12 33 32 PM

**Review by aditya-mitra — APPROVED** [review]

**Pull request merged**